tests/models/siglip/test_modeling_siglip.py

```diff
@@ -587,6 +799,104 @@ def test_model_from_pretrained(self):
         model = SiglipModel.from_pretrained(model_name)
         self.assertIsNotNone(model)
 
+    @require_flash_attn
+    @require_torch_gpu
+    @mark.flash_attn_test
+    @slow
+    @is_flaky()
```

Comment:
Is it flaky for siglip? Do we know why? I know we have this decorator for the common tests, but not for the model-specific implementations